pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml

```diff
@@ -0,0 +1,191 @@
+values:
+  central:
+    persistence:
+      none: true
+  ca:
+    cert: ""
+    key: ""
+  scannerV4:
+    indexer:
+      serviceTLS:
+        cert: ""
+        key: ""
+    matcher:
+      serviceTLS:
+        cert: ""
+        key: ""
+    db:
+      serviceTLS:
+        cert: ""
+        key: ""
+  monitoring:
+    openshift:
+      enabled: false
+server:
+  visibleSchemas:
+  - openshift-4.1.0
```

Comment:
curious how much does this version matter? for example, we are on openshift 4.14 at this point